### DISCUSSION TITLE
Feature Request: Playground should have export and import of a model, tuples and assertions

### DISCUSSION THREAD

**Discussion by ghstahl** [openfga/community#18] [Ideas]
It is easy enough to copy the authorization model, but I can't copy the tuples or assertions.

It would be nice to export the entire store, within reason, when modeling.  i.e. put a limit on the number or tuples and assertions allowed.

This would let me import it into a developer docker instance of openFGA 

The import should also be exposed as an api so I can seed openFGA easily for unit tests and development.

Also, it would be nice to in line edit tuples and assertions.
Right now, its delete and replace.

**Comment by rhamzeh** [comment]
Thanks for the feedback @ghstahl!

Created a separate discussion [here](https://github.com/openfga/community/discussions/19#discussion-4287911) for allowing editing tuples.

**Comment by rhamzeh** [comment]
Just as a follow-up because we did not post here when we enabled the export feature last year.

On the Playground, you can press the overflow () button on the top right corner and select `Copy Store Data`. The data will be copied as a JSON string and contain the store id, name, latest authorization model, tuples and assertions.

<img width="792" alt="Screen Shot 2023-02-08 at 11 23 28 AM" src="https://user-images.githubusercontent.com/536147/217589936-72c5e297-9678-47c1-b7eb-3399b1526bee.png">
